.storybook/decorators/withHits.ts

```diff
@@ -26,94 +26,98 @@ type SearchOptions = Omit<
     playground?: Playground;
   };
 
-export const withHits = (
-  storyFn: ({
-    container,
-    instantsearch,
-    search,
-  }: {
-    container: HTMLElement;
-    instantsearch: InstantSearchUMDModule;
-    search: InstantSearch;
-  }) => void,
-  searchOptions?: SearchOptions
-) => () => {
-  const {
-    appId = 'latency',
-    apiKey = '6be0576ff61c053d5f9a3225e2a90f76',
-    indexName = 'instant_search',
-    playground = defaultPlayground,
-    ...instantsearchOptions
-  } = searchOptions || {};
-
-  const urlLogger = action('Routing state');
-  const search = instantsearch({
-    indexName,
-    searchClient: algoliasearch(appId, apiKey),
-    routing: {
-      router: {
-        write: (routeState: object) => {
-          urlLogger(JSON.stringify(routeState, null, 2));
+export const withHits =
```

Comment:
this file just hasn't been reformatted with prettier in a while I think, not sure how that happened, but only change is moving searchClient outside so it can be monkeypatched easier for debugging

Reply:
With whitespace diff off its more manageable to read 👍 .